extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -54,6 +54,7 @@ public static function onInstantGlobalsGetVariables( array &$vars )
 		$vars[] = 'wgAdDriverSourcePointCountries';
```

Comment:
I'm thinking about marking all the places where we're keeping current variables which we want to remove in future. We could mark them with a comment with ADEN ticket, what do you think about it?